stac_fastapi/types/stac_fastapi/types/config.py

```diff
@@ -27,14 +29,29 @@ class ApiSettings(BaseSettings):
     app_host: str = "0.0.0.0"
     app_port: int = 8000
     reload: bool = True
+
+    # Enable Pydantic validation for output Response
     enable_response_models: bool = False
 
+    # Enable direct `Response` from endpoint, skipping validation and serialization
+    enable_direct_response: bool = False
```

Comment:
any opinions on the option's name?

Reply:
LGTM

Reply:
You don't want to enable direct response by default? It's a new change, better to have it false.

Reply:
True makes sense